superset-frontend/src/components/Modal/Modal.tsx

```diff
@@ -46,6 +53,10 @@ export interface ModalProps {
   wrapProps?: object;
   height?: string;
   closable?: boolean;
+  resizable?: boolean;
+  resizableConfig?: ResizableProps;
+  draggable?: boolean;
+  draggableConfig?: DraggableProps;
```

Comment:
It feels slightly redundant to have four props for essentially two features. Would it be possible to squeeze the `resizable` into `ResizableProps` as an `enabled` prop which is assumed to default to `false`? Same for `draggable`.

Reply:
I would say that `resizableConfig` and `draggableConfig` might be a bit overkilling here and the only reason why I put them there is that I cannot predict right now how many of those options we will need in the future. We might as well realize we don't need any and just shut them down.

So rather than squeezing `resizable` and `draggable` inside the config which looks like very standard props, I would first implement these types of Modals in a few places and then either shut the config props down altogether or just come up with a few elected ones as overridable defaults. What do you think?

Reply:
Yes, that works just fine. The reason I pointed this out is this prop structure looked "unexpected" (for the lack of a better word), but I assumed there was a good reason for it. I completely agree; let's get this merged soon and do many small iterative improvements later rather than one big PR now 👍